spec/std/openssl/rsa_spec.cr

```diff
@@ -0,0 +1,128 @@
+require "spec"
+require "openssl/rsa"
+
+describe OpenSSL::RSA do
+  describe "instantiating and generate a key" do
+    it "can instantiate and generate for a given key size" do
+      pkey = OpenSSL::RSA.new(256)
+      pkey.private?.should be_true
+      pkey.public?.should be_false
+
+      pkey.public_key.public?.should be_true
+    end
+    it "can export to PEM format" do
```

Comment:
Typical style is to treat `it` blocks like `def` methods and leave a line between each.